src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupUtilities.cs

```diff
@@ -16,11 +16,12 @@
 
 namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery
 {
-    public class BackupUtilities
+    public class BackupUtilities: IBackupUtilities
     {
         private CDataContainer dataContainer;
         private ServerConnection serverConnection;
-        private CommonUtilities backupRestoreUtil = null;        
+        private CommonUtilities backupRestoreUtil = null;
+        private Backup bk = null;
```

Comment:
>bk [](start = 23, length = 2)

please use a more complete name for this variable that follows code base conventions.  For example, "backup"